docs/tags.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: sub-navigation
+order: 7
+title: Adding tags
+description: Use tags to categorise posts and make it easier for readers to browse content on your site.
+---
+
+The GOV.UK Eleventy Plugin makes it easy to add tags to your website. Follow these instructions to enable this feature.
```

Comment:
Not sure we need to say it’s "easy" (and maybe it’s not _that_ easy given need to create 2 files)?

```suggestion
The GOV.UK Eleventy Plugin lets you use tags to categorise pages. Each post can display its list of tags, which link to a page that lists other posts with the same tag. Follow these instructions to enable this feature.
```